roles/setup-slack/README.md

```diff
@@ -0,0 +1,56 @@
+Setup Slack Team
+=========
+
+An ansible role to setup slack team.
+
+
+Role Variables
+--------------
+
+This is a sample of a YAML file containing the variables.
+
+```
+---
+slack_token: XXXXXXXXXXX
+slack_users:
+  - first_name: AAA
+    last_name: BBBB
+    email: aaabbb@example.com
+    channels:
+      - "a"
+      - "b"
+      - "c"
+slack_channels:
+  - name: "c"
+    private: no
+```
+
+Here are the description of each variables:
+
+- `slack_token`: The token generated by a user that has admin priviledge (see https://get.slack.help/hc/en-us/articles/215770388-Create-and-regenerate-API-tokens for instructions)
+- `slack_users`: a list of dictionaries with user data
+- `slack_channels`: a list of groups to be created (can be an empty list)
+
+Due to the sensitive nature of the variables it's best to use ansible-vault to create the `vars.yml`
+
+Steps:
+1. `$ ansible-vault create vars.yml` it will ask for a password and it will be needed when we're running the playbook later
+2. Specify the variables in the file and save it
+
+
+**Important note:** 
+* You must be running this as an Admin user in the Slack workspace.
+* If a username includes special characters (including spaces) it must be URL encoded.
```

Comment:
Instead of requiring the user to take care of this, I'd say let's use the `urlencode` filter to handle it - i.e.: use 
```
{{ user.first_name | urlencode }}
{{ user.last_name | urlencode }}
```